vickyctl/src/main.rs

```diff
@@ -61,11 +62,23 @@ struct TasksArgs {
     ctx: AppContext,
 }
 
+#[derive(Args, Debug)]
+#[command(version, about = "Show all poisoned locks vicky is managing", long_about = None)]
+struct LocksArgs {
+    #[command(flatten)]
+    ctx: AppContext,
+    #[clap(long)]
```

Comment:
active and poisoned should not be a option but an argument.

Therefore valid commands would be `... locks active` and not `... locks --active`.
This eliminates the needed validation that at least `active` or `poisoned` is given as argument, which is currently missing.

Reply:
Well, we don't really care for active locks all that much. So poisoned locks is the default, which is why this is the case. If no argument is supplied, it shows poisoned locks. And to me, active or poisoned are properties of locks, therefore I made them arguments.